javascript/567-Permutation-In-String.js

```diff
@@ -1,158 +1,68 @@
-//////////////////////////////////////////////////////////////////////////////
-// Static Sliding Window
-// Time: Theta(l1 + l2) O(l1 + l2)  Space: Theta(1) O(1)
-// Highest performing solution. Simply builds a map of the character counts
-// for `s1` and `s1.length` of `s2` whose characters are within `s1`, updates
-// the `s2` character map as it slides from the beginning of `s2` to the end
-// of `s2`, and returns upon verifying a match between the `s1` and `s2`
-// character maps.
-//////////////////////////////////////////////////////////////////////////////
-
 /**
  * @param {string} s1
  * @param {string} s2
+ * Time O(N + (M - N)) | Space O(1)
  * @return {boolean}
  */
-function checkInclusion(s1, s2) {
-    
-    if (s1.length > s2.length) {
-        return false;
-    }
-    
-    const s1Chars = Object.create(null);
-    const s2Chars = Object.create(null);
-    
-    for (const ch of s1) {
-        if (!(ch in s1Chars)) {
-            s1Chars[ch] = 0;
-            s2Chars[ch] = 0;
-        }
-        ++s1Chars[ch];
-    }
+var checkInclusion = (s1, s2) => {
```

Comment:
i love the readability changes 💯 

i'd also love to keep both of the former solutions (because each is instructive in a helpful way) included. do you mind keeping both, and keeping the parts of the comments that explain how they're distinct?